src/components/ErrorComponents/errorParser.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * This hook will take an error caught from a promise,
+ * and attempt to parse it and retrieve any message, code, and stacktrace
+ */
+
+// Passing in the intl object isn't ideal,
+// but this can't be a hook if it's going to run conditionally
+const errorParser = async (error, intl) => {
+  let errorMessage;
+  let errorBody;
+
+  /* For HTTPError we assume error.response returns a Promise */
+  if (error.name.toLowerCase() === 'httperror') {
```

Comment:
Are we sure there would be an `error.name`? If not this line is going to crash. Use optional changing instead. once destructured, you could do 
```suggestion
  if (name?.toLowerCase() === 'httperror') {
```